go-controller/pkg/ovn/obj_retry.go

```diff
@@ -300,15 +323,23 @@ func areResourcesEqual(objType reflect.Type, obj1, obj2 interface{}) (bool, erro
 		return true, nil
 
 	case factory.EgressFirewallType:
-		newEgressFirewall, ok := obj1.(*egressfirewall.EgressFirewall)
+		oldEgressFirewall, ok := obj1.(*egressfirewall.EgressFirewall)
```

Comment:
oops :)

Reply:
in the end it didn't matter, since we're just checking if the two objects are equal :)

Reply:
yeah just wanted to keep aligned